src/plugins/biostruct3d_view/src/BioStruct3DGLRender.h

```diff
@@ -152,27 +150,19 @@ class BioStruct3DGLRenderer {
     void setColorScheme(const BioStruct3DColorScheme *s);
 
     //! @returns current color scheme.
-    const BioStruct3DColorScheme *getColorScheme() const {
-        return colorScheme;
-    }
+    const BioStruct3DColorScheme *getColorScheme() const;
```

Comment:
The definition is lost. Also getShownModelsIndexes, setShownModelsIndexes, isAvailableFor

Reply:
Good job!